openid-4-verifiable-presentations-1_0.md

```diff
@@ -232,9 +232,9 @@ Presentation of Verifiable Credentials using OpenID for Verifiable Presentations
 
 The Authorization Request follows the definition given in [@!RFC6749] taking into account the recommendations given in [@!I-D.ietf-oauth-security-topics].
 
-The Verifier may send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101].
+The Verifier MAY send an Authorization Request as a Request Object either by value or by reference, as defined in the JWT-Secured Authorization Request (JAR) [@RFC9101]. The Request Object MAY contain a subset of parameters, which the Wallet uses to request the creation of a new Request Object from the Verifier. This request is made using an HTTP POST to the Verifier's `request_uri` endpoint. The Wallet MAY support this feature, which involves providing some details about its technical capabilities with the Verifier. This allows the Verifier to generate a Request Object that is specifically designed to match the capabilities of the Wallet making the request. Verifiers that support this feature MUST provide the `request_uri_method`, as defined below, to indicate to the Wallet that they support this feature.
 
-The Verifier articulates requirements of the Credential(s) that are requested using `presentation_definition` and `presentation_definition_uri` parameters that contain a Presentation Definition JSON object as defined in Section 5 of [@!DIF.PresentationExchange]. Wallet implementations MUST process Presentation Definition JSON object and select candidate Verifiable Credential(s) using the evaluation process described in Section 8 of [@!DIF.PresentationExchange].
+The Verifier specifies the requirements for the Credential(s) being requested using the `presentation_definition` and `presentation_definition_uri` parameters. These parameters contain a Presentation Definition JSON object as outlined in Section 5 of [@!DIF.PresentationExchange]. Wallet implementations are required to process this Presentation Definition JSON object and select suitable Verifiable Credential(s) following the evaluation process detailed in Section 8 of [@!DIF.PresentationExchange].
```

Comment:
```suggestion
The Verifier specifies the requirements for the Credential(s) being requested using the `presentation_definition` or `presentation_definition_uri` parameters. These parameters contain a Presentation Definition JSON object as outlined in Section 5 of [@!DIF.PresentationExchange]. Wallet implementations are required to process this Presentation Definition JSON object and select suitable Verifiable Credential(s) following the evaluation process detailed in Section 8 of [@!DIF.PresentationExchange].
```
I presume the parameters are mutually exclusive?
Also, in case of the `...uri`, the P.D. JSON is not contained in the parameter.